facturx/facturx.py

```diff
@@ -147,3 +147,31 @@ def write_xml(self, path):
         with open(path, 'wb') as f:
             f.write(self.xml_str)
 
+    def __make_dict(self):
+        fields_data = xml_flavor.FIELDS
+        flavor = self.flavor.name
+
+        output_dict = None
+        for field in fields_data.keys():
+            if fields_data[field]['_required']:
```

Comment:
Why are non-required fields not exported?

Reply:
Yes, it should export non-required fields too.